### PR TITLE
 [RFC] API: nvim_input('<c-Ñ>') unrecognized

### PR DESCRIPTION
**Pull request by MichaHoffmann**
Started working on #8343

### CODE REVIEW

**Comment by justinmk** [comment]
That makes sense.

File: src/nvim/keymap.c [hunk 1]
```diff
@@ -582,7 +581,7 @@ int find_special_key(const char_u **srcp, const size_t src_len, int *const modp,
         // Anything accepted, like <C-?>.
         // <C-"> or <M-"> are not special in strings as " is
         // the string delimiter. With a backslash it works: <M-\">
-        if (end - bp > l && !(in_string && bp[1] == '"') && bp[2] == '>') {
+        if (end - bp > l && !(in_string && bp[1] == '"') && bp[l+1] == '>') {
```
Comment (justinmk) [inline src/nvim/keymap.c]: I wonder why cb02137dfac7357650a2e9cc32acb66326e59058 changed it to `bp[2]`. @jamessan what do you think?

Comment (jamessan) [reply src/nvim/keymap.c]: Because that's what upstream did (and still does).  Cc @brammool and @chrisbra.

**Comment by jamessan** [comment]
This was fixed by #10794

**Pull request closed without merging**